ravenframework/CodeInterfaceClasses/MFIX/mfixInterface.py

```diff
@@ -0,0 +1,504 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
```

Comment:
`ravenframework/CodeInterfaceClasses/MFIX/` needs to have a `__init__.py` file added.

Reply:
@joshua-cogliati-inl I added empty __init__.py file.

Reply:
Thank you.